homeassistant/components/smlight/button.py

```diff
@@ -0,0 +1,86 @@
+"""Support for SLZB-06 buttons."""
+
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+import logging
+from typing import Final
+
+from pysmlight.web import CmdWrapper
+
+from homeassistant.components.button import (
+    ButtonDeviceClass,
+    ButtonEntity,
+    ButtonEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .coordinator import SmDataUpdateCoordinator
+from .entity import SmEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class SmButtonDescription(ButtonEntityDescription):
+    """Class to describe a Button entity."""
+
+    entity_category = EntityCategory.CONFIG
```

Comment:
set it to `_attr_entity_category` in the class

Reply:
ok